tests/providers/snowflake/hooks/test_snowflake.py

```diff
@@ -388,3 +388,38 @@ def test_connection_failure(self, mock_run):
             assert status is False
             assert msg == 'Connection Errors'
             mock_run.assert_called_once_with(sql='select 1')
+
+    @mock.patch('airflow.providers.snowflake.hooks.snowflake.SnowflakeHook.run')
```

Comment:
Is it needed?

Reply:
@mik-laj  Just saw that you modified the pytest connection parameters to allow a blank value for region. I attempted to write the same, but unsuccessfully, so added a new test. This is not needed, now that you have added the params.